src/com/sun/ts/tests/compat13/ejb/tx/TestBeanEJB.java

```diff
@@ -26,7 +26,7 @@
 import java.util.*;
 import java.rmi.*;
 import jakarta.ejb.*;
-import javax.transaction.*;
```

Comment:
Haha, I see that I missed all of the import javax.transaction.*;  If we need to reference any of the javax.transaction.xa classes, we might need to add them individually.  

I used the following to only update the specific transaction references:
`
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.H/jakarta\.transaction\.H/g'
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.I/jakarta\.transaction\.I/g'
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.N/jakarta\.transaction\.N/g'
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.R/jakarta\.transaction\.R/g'
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.S/jakarta\.transaction\.S/g'
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.T/jakarta\.transaction\.T/g'
ag --ignore .git  -l javax.transaction.| xargs sed -i 's/javax\.transaction\.U/jakarta\.transaction\.U/g'
`

Reply:
Yes,  javax.transaction.xa is a problem.
Also looks that I hat a rerason for some of the jakarta.transaction -> javax.transaction rollbacks.
Now we have one build issue back:
` [ts.javac] /data/jakartaee-tk/src/com/sun/ts/tests/common/connector/whitebox/TestBootstrapContext.java:85: error: incompatible types: javax.transaction.TransactionSynchronizationRegistry cannot be converted to jakarta.transaction.TransactionSynchronizationRegistry`

But it may be fixed later. I don't want to make this PR even bigger.

Reply:
Anyway, if there is a problem with  javax.transaction.xa, you will see some `Class not found` errors. And there were not seen in this part of the code so it shall be ok.